src/cli.ts

```diff
@@ -29,6 +29,12 @@ const _ = yargs(hideBin(process.argv))
     description: "The EVM compatible chain to target",
     default: "ethereum-goerli",
   })
+  .option("r", {
```

Comment:
We enable this, but don't support it fully yet. Didn't want to get this in the way of the rest of this PR.